### ISSUE TITLE
[FEATURE] Make binds save per config

### ISSUE THREAD

**Issue by SuicidalEeper** [CCBlueX/LiquidBounce#4073]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

Same reasoning as in my previous issue, binds should be saved per config (or at least have an option to) because on some configs I bind a bunch of stuff I might not wanna toggle on another config

**Comment by notKnownn** [comment]
Yes please, this would be awesome (it was possible in legacy, why not on next gen? Feels like a downgrade)

**Comment by 1zun4** [comment]
Already possible. .localsettings create name binds
